test/v1/connectors/ListBenefitsConnectorSpec.scala

```diff
@@ -61,7 +61,7 @@ class ListBenefitsConnectorSpec extends ConnectorSpec {
     )
   )
 
-  class Test extends MockHttpClient with MockAppConfig {
+  trait Test extends MockHttpClient with MockAppConfig {
```

Comment:
Can we move this out the way to the bottom of the file?

Reply:
Sure. Also shifted the order around in DeleteBenefitAmountsConnectorSpec for consistency